### DISCUSSION TITLE
1.16.5 - Nightly: removed blocks, fix water placing in nether, changed piston head waterlogging, fixed water flowing sounds

### DISCUSSION THREAD

**Discussion by Felicis** [Felicis/Water-Physics-Overhaul#33] [Nightly Releases]
THIS VERSION IS NIGHTLY: backup your worlds, expect crashes! Do not post issues for this, instead join the discussion :)

Hi everyone,

here's another nightly, for who is interested in the bleeding edge :P

Here's the changelog (this time also tested with lava):

- old blocks and blockentities removed (WPO fluid pump, fluid gate, fluid pipe)
- water can not be placed in nether anymore
- water flowing sounds fixed
- piston head can now be fluidlogged, but when moving it expels the contained fluid

<hr /><em>This discussion was created from the release <a href='https://github.com/Felicis/Water-Physics-Overhaul/releases/tag/0.1.3-alpha4'>1.16.5 - Nightly: removed blocks, fix water placing in nether, changed piston head waterlogging, fixed water flowing sounds</a>.</em>

**Comment by saerdodus** [comment]
Looking forward to your continued development, my friend and I are on 1.18.2 waiting for it!

**Comment by vrgamer123** [comment]
are you still working on the 1.12 version?

**Comment by Felicis** [comment]
I haven't even started on the 1.12 version yet. It's in line after finishing the 1.16 version, and upgrading it to 1.17 and 1.18 (because upgrading is easier than downgrading). So it will take a while, if I manage to get there at all :|
------- Original Message -------
